include/experimental/__p1673_bits/blas1_vector_idx_abs_max.hpp

```diff
@@ -114,7 +124,7 @@ typename extents<>::size_type idx_abs_max(
   std::experimental::mdspan<ElementType, std::experimental::extents<ext0>, Layout, Accessor> v)
 {
 
-  // if vector is empty, always retun according to our proposal
+  // if vector is empty, always return according to our proposal
```

Comment:
Nice catch! : - )

Reply:
you pointed it out in the other PR :) so good catch you!